common.yml

```diff
@@ -18,6 +18,7 @@ services:
       - PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME_REVISION
       - PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME
       - PFB_TILEGARDEN_ROOT=http://localhost:9400
+      - PFB_TILEGARDEN_CACHE_BUCKET=dev-pfb-tilecache-us-east-1
```

Comment:
Why not put the tiles for development users somewhere in the existing `${DEV_USER}-pfb-storage-us-east-1` bucket we're using for other S3 storage?

Reply:
I'm expecting that for serving tiles from the cache we'll want to use an [S3 Website](https://docs.aws.amazon.com/AmazonS3/latest/dev/HowDoIWebsiteConfiguration.html), since that's what I did in [the Tilegarden branch](https://github.com/azavea/tilegarden/tree/feature/kjh/s3-tile-cache) where I've been working on caching.  And as far as I can tell that serves the whole bucket.  If there turns out to be a way to restrict it and use the existing bucket, that would be worth pursuing.

It's true that a shared dev cache could cause some inconvenience, though given that tiles are namespaced by job ID, it would be more in the way of clutter than any weird tiler behavior.  I'm inclined to run with it, though--with the fallback of making a personal bucket and overriding the variable if one is needed--until we know whether it's a problem in practice.